tests/test_ssl_context_adapter.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""Tests for the SSLContextAdapter."""
+
+import pickle
+from ssl import SSLContext, PROTOCOL_TLSv1
+
+import mock
+import pytest
+import requests
+from requests.adapters import HTTPAdapter
+from requests_toolbelt.adapters.ssl_context import SSLContextAdapter
+
+
+@pytest.mark.skipif(requests.__build__ < 0x020400,
+                    reason="Test case for newer requests versions.")
+@mock.patch.object(HTTPAdapter, 'init_poolmanager')
+def test_ssl_context_arg_is_passed_on_newer_requests(init_poolmanager):
```

Comment:
Please add docstrings to the test functions

Reply:
See also my comment on docstrings in the actual module - I've added slightly wordier descriptions as docstrings now. Is that what you expected @sigmavirus24?